api/src/main/resources/org/openmrs/api/db/hibernate/OrderSet.hbm.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+
+    <class name="OrderSet" table="order_set">
+
+        <id name="orderSetId" type="int" column="order_set_id" unsaved-value="0">
+            <generator class="native">
+                <param name="sequence">order_set_order_set_id_seq</param>
+            </generator>
+        </id>
+
+        <discriminator column="order_set_id" insert="false" />
+
+        <property name="name" type="java.lang.String"
+                  column="name" not-null="true" />
+        <property name="description" type="java.lang.String"
+                  column="description" not-null="true" />
+        <property name="operator" column="operator" not-null="true">
+            <type name="org.openmrs.util.HibernateEnumType">
+                <param name="enumClassName">org.openmrs.OrderSet$Operator</param>
+            </type>
+        </property>
+
+        <list name="orderSetMembers"  lazy="true" cascade="all-delete-orphan" access="field">
+            <key column="order_set_id" not-null="true"/>
+            <list-index column="sort_weight"/>
```

Comment:
If list-index has to be an integer then i guess we shouldn't use a list-index and instead add a sortWeight based Comparator that is used in OrderGroup.getOrders() and OrderSet.getSetMembers()